common/core/desktop/tests/unit/kmx/meson.build

```diff
@@ -77,7 +77,7 @@ tests = [
 ]
 
 if build_machine.system() == 'windows'
-  kmcomp = find_program('kmcomp.exe', required: false)
+  kmcomp = find_program(join_paths(meson.source_root(),'..','..','..','windows','bin','developer','kmcomp.exe'), 'kmcomp.exe', required: false)
```

Comment:
Given your answer to my earlier comment...

> I wanted to make sure I was using the right kmcomp.exe when building unit tests for Keyman Core.

I didn't quite get that at the time, as the way it was worded made me think that there wasn't a related change here.  Looking at it again, though, I think the quote above relates to this line?

Reply:
yep